pkg/cluster/resources.go

```diff
@@ -154,6 +154,35 @@ func CreateNamespace(ctx context.Context, cli client.Client, namespace string, m
 	return desiredNamespace, client.IgnoreAlreadyExists(createErr)
 }
 
+// ProcessAllNamespace execute func() on all active namespaces in the cluster with 500 as chunksize.
+func ProcessAllNamespace(ctx context.Context, cli client.Client, processFunc func(*corev1.Namespace) error) error {
+	namespaceList := &corev1.NamespaceList{}
```

Comment:
Why not `namespaces`?

Reply:
only followed the pattern name List from WaitForDeploymentAvailable() but can update to namespaces